thunderbird/processes/wps_generate_prsn.py

```diff
@@ -152,15 +152,10 @@ def _handler(self, request, response):
             generate_prsn_file(filepaths, chunk_size, self.workdir, output_file)
 
             response.update_status("File processing complete", 90)
-            prsn_file = collect_output_files("prsn", self.workdir)
+            prsn_file = collect_output_files("prsn", self.workdir)[0]
```

Comment:
Is it possible to just use the `,` here instead of the index? Like so:
```python3
prsn_file, = collect_output_files("prsn", self.workdir)
```

Reply:
That works.